Mars Age

PrasantiNalla opened the pull request:
mars age calculator added to birthday page. 
![image](https://user-images.githubusercontent.com/121449344/219646138-fa229e3c-ed3d-4224-9a1a-af557af990a5.png)
![image](https://user-images.githubusercontent.com/121449344/219646213-bcdd31fa-4d47-4d0a-979b-24004302eb5a.png)

PrasantiNalla committed:
Mars Age

mars age calculator added to birthday page

francescastocco commented on src/components/Birthday/Birthday.tsx:
```diff
@@ -1,23 +1,38 @@
-            <div>IMPORT PDF GENERATOR COMPONENT</div>
+            <DateNameUserInput setName={setName} setBirthday={setBirthday} handleSubmit={handleSubmit} />
+            {/* <div>IMPORT DROPDOWN YEAR SELECTION COMPONENT</div> */}
+            <MarsViewer />
```
I think here we just want the ImageSelector & ImageViewer components so we can pass the birthday date

francescastocco reviewed (commented):
Looks good, just a small thing

PrasantiNalla committed:
ImageSelector & ImageViewer components

 ImageSelector & ImageViewer components have been added and have been commented out as the latest code doesn't have the functionality implemented yet.

PrasantiNalla committed:
Merge branch 'staging' into mm-3.5-MarsAgeCalculator
